src/components/dataDisplays/RelationshipDisplay.jsx

```diff
@@ -43,6 +43,7 @@ export default function RelationshipDisplay({
 
   return (
     <DataDisplay
+      idKey="guid"
```

Comment:
The relationships should have guids. That's just a fallback to keep the page from crashing, right?

Reply:
It makes it an unreliable key though. Wouldn't it be better to filter out relationships without GUIDs, and then uses the GIUD as the key, knowing that it exists?

Reply:
As far as we know, we can rely on relationships having guids. What we dont want to do is crash the page completely if a relationship is missing a guid, because that makes the whole page unusable. But we are very much okay with getting a key error or a display issue because we are not expecting this to come up, and it would be a clue to investigate if it ever does.